projects/packages/masterbar/src/admin-menu/class-jetpack-admin-menu.php

```diff
@@ -70,9 +71,11 @@ public function get_preferred_view( $screen, $fallback_global_preference = true
 	 */
 	public function get_cpt_menu_link( $ptype_obj ) {
 
-		$post_type = $ptype_obj->name;
+		$post_type                          = $ptype_obj->name;
+		$is_self_hosted_site                = ! ( new Host() )->is_wpcom_platform();
```

Comment:
Do we need that added check here? Looking at the code, `Jetpack_Admin_Menu` should only be loaded on self-hosted sites already:

https://github.com/Automattic/jetpack/blob/15de818749232d2c11e2cf9b7650c558119528ed/projects/packages/masterbar/src/admin-menu/load.php#L130-L132

Reply:
Thanks for the feedback @jeherve! I updated the code and tested it.